samples/all.sh

```diff
@@ -41,10 +41,8 @@ $(DMD) winsamp gdi32.lib winsamp.def
 winsamp
 
 # COM client/server example
-$(DMD) -c dserver -release $(DFLAGS)
-$(DMD) -c chello $(DFLAGS)
-$(DMD) dserver.obj chello.obj uuid.lib ole32.lib advapi32.lib kernel32.lib user32.lib dserver.def
-#dclient will faill unless run with administrator rights
+$(DMD) dserver.d chello.d dserver.def advapi32.lib ole32.lib user32.lib
+# dclient will fail unless run with administrator rights
```

Comment:
why remove dserver.def? .def files may not be needed for this simple project, but for more complex ones it can be needed. Hence a sample showing how they are used.